File: HtmlEditableContent/Interop/src/markupvalue.ts
import { MarkUpRange } from "./markuprange";

export class MarkUpValue {
    Init(id: string, content: string, html: string, range: MarkUpRange): void {
        this.id = id;
        this.value = content;
        this.innerHtml = html;
        this.range = range;
    }
    innerHtml: string
    id: string;
    range: MarkUpRange;
    value: string;
}

File: HtmlEditableContent/Interop/MarkUpValue.cs
using HtmlBuilder;

namespace HtmlEditableContent.Interop
{
    public class MarkUpValue
    {
        public MarkUpValue() { }
        public MarkUpRange Range { get; set; }
        public string Id { get; set; }
        public string Value { get; set; }
        public string InnerHtml { get; set; }
    }
}
